### PR TITLE
Docs: Add documentation to the SDK to resolve MD warnings

### PR DESCRIPTION
**Pull request by nickybondarenko**
This PR is adding documentation placeholders that resolve most of the issues in the generated API reports.

### CODE REVIEW

**Review by fabriziodemaria — DISMISSED** [review]

File: packages/react/src/index.tsx [hunk 1]
```diff
@@ -87,33 +109,44 @@ export class ConfidenceReact implements EventSender, Trackable, FlagResolver {
...
   }
 
   /* eslint-disable react-hooks/rules-of-hooks */
 
+  /** Uses Context */
   useContext(): Context {
     this.assertContext('useContext', 'getContext');
     return useConfidenceContext(this);
   }
+  /** Uses Context */
```
Comment (nicklasl) [inline packages/react/src/index.tsx:139]:
```suggestion
  /** Hook to access the WithContext functionality. Returns a  ConfidenceReact instance with the passed context. */
```

File: packages/react/src/index.tsx [hunk 2]
```diff
@@ -87,33 +109,44 @@ export class ConfidenceReact implements EventSender, Trackable, FlagResolver {
...
   useWithContext(context: Context): ConfidenceReact {
     this.assertContext('useWithContext', 'withContext');
     return useWithContext(context, this);
   }
+  /** Uses EvaluateFlag */
```
Comment (nicklasl) [inline packages/react/src/index.tsx:144]:
```suggestion
  /** Hook to use EvaluateFlag functionality */
```

File: packages/react/src/index.tsx [hunk 3]
```diff
@@ -87,33 +109,44 @@ export class ConfidenceReact implements EventSender, Trackable, FlagResolver {
...
   useEvaluateFlag<T extends Value>(path: string, defaultValue: T): FlagEvaluation<Value.Widen<T>> {
     this.assertContext('useEvaluateFlag', 'evaluateFlag');
     return useEvaluateFlag(path, defaultValue, this);
   }
+  /** Uses Flag */
```
Comment (nicklasl) [inline packages/react/src/index.tsx:149]:
```suggestion
  /** Hook to use getFlag functionality */
```

File: packages/sdk/src/Confidence.ts [hunk 4]
```diff
@@ -147,15 +178,34 @@ export class Confidence implements EventSender, Trackable, FlagResolver {
...
     return child;
   }
 
+  /**
+   * Tracks an event
+   * @param name - event name
+   * @param data - data to track */
   track(name: string, data?: EventData): void;
+  /**
+   * Tracks an event
```
Comment (nicklasl) [inline packages/sdk/src/Confidence.ts:199]:
```suggestion
   * Sets up a Trackable.Manager to manage event tracking or context changes.
```

File: packages/openfeature-server-provider/package.json [hunk 5]
```diff
@@ -8,12 +8,12 @@
...
     "@openfeature/core": "^1.1.0",
     "@openfeature/server-sdk": "^1.13.5",
-    "@spotify-confidence/sdk": "0.1.4",
+    "@spotify-confidence/sdk": "0.1.5",
     "rollup": "4.14.2"
   },
   "peerDependencies": {
     "@openfeature/server-sdk": "^1.13.5",
-    "@spotify-confidence/sdk": "^0.1.4"
+    "@spotify-confidence/sdk": "^0.1.5"
```
[the remark this answers is not in the document]
Comment (nickybondarenko) [reply packages/openfeature-server-provider/package.json:16]: How did that get there 😓

**Review by nicklasl — APPROVED** [review]

**Pull request merged**